zigbee2mqtt-edge/config.json

```diff
@@ -35,9 +35,9 @@
     "log_level": "match(^info|debug|warn|error$)?",
     "rtscts": "bool?",
     "include_device_information": "bool?",
+    "soft_reset_timeout": "int?",
```

Comment:
I moved this up, so that it was in the same position in both edge and non-edge config JSON's - making it easier to compare and spot any missing options..